stations/views.py

```diff
@@ -66,22 +88,19 @@ def bikes(self, request, *args, **kwargs):  # return bikes
                 },
                 status=status.HTTP_422_UNPROCESSABLE_ENTITY,
             )
-        # TODO(kkrolik): uncomment once capasity of station is introduced
```

Comment:
xD angielski 10/10

Reply:
mi PyCharm podświetla, tak to bym nie wiedział